src/legacy/core_plugins/kibana/public/dashboard/listing/dashboard_listing.js

```diff
@@ -54,271 +40,59 @@ class DashboardListingUi extends React.Component {
-          onCancel={this.closeDeleteModal}
-          onConfirm={this.deleteSelectedItems}
-          cancelButtonText={
-            <FormattedMessage
-              id="kbn.dashboard.listing.deleteSelectedDashboardsConfirmModal.cancelButtonLabel"
-              defaultMessage="Cancel"
-            />
-          }
-          confirmButtonText={
-            <FormattedMessage
-              id="kbn.dashboard.listing.deleteSelectedDashboardsConfirmModal.confirmButtonLabel"
-              defaultMessage="Delete"
-            />
-          }
-          defaultFocusedButton="cancel"
-        >
-          <p>
-            <FormattedMessage
-              id="kbn.dashboard.listing.deleteDashboardsConfirmModalDescription"
-              defaultMessage="You can't recover deleted dashboards."
-            />
-          </p>
-        </EuiConfirmModal>
-      </EuiOverlayMask>
-    );
-  }
-
-  renderListingLimitWarning() {
-    if (this.state.showLimitError) {
-      return (
-        <React.Fragment>
-          <EuiCallOut
-            title={
-              <FormattedMessage
-                id="kbn.dashboard.listing.listingLimitExceededTitle"
-                defaultMessage="Listing limit exceeded"
-              />
-            }
-            color="warning"
-            iconType="help"
-          >
-            <p>
-              <FormattedMessage
-                id="kbn.dashboard.listing.listingLimitExceededDescription"
-                defaultMessage="You have {totalDashboards} dashboards, but your {listingLimitText} setting prevents
-                the table below from displaying more than {listingLimitValue}. You can change this setting under {advancedSettingsLink}."
-                values={{
-                  totalDashboards: this.state.totalDashboards,
-                  listingLimitValue: this.props.listingLimit,
-                  listingLimitText: (
-                    <strong>
-                      listingLimit
-                    </strong>
-                  ),
-                  advancedSettingsLink: (
-                    <EuiLink href="#/management/kibana/settings">
-                      <FormattedMessage
-                        id="kbn.dashboard.listing.listingLimitExceeded.advancedSettingsLinkText"
-                        defaultMessage="Advanced Settings"
-                      />
-                    </EuiLink>
-                  )
-                }}
-              />
-            </p>
-          </EuiCallOut>
-          <EuiSpacer size="m" />
-        </React.Fragment>
-      );
-    }
   }
 
-  renderNoResultsMessage() {
-    if (this.state.isFetchingItems) {
-      return '';
-    }
-
+  render() {
+    const { intl } = this.props;
     return (
-      <FormattedMessage
-        id="kbn.dashboard.listing.noMatchedDashboardsMessage"
-        defaultMessage="No dashboards matched your search."
+      <TableListView
+        create={this.props.create}
+        find={this.props.find}
+        delete={this.props.delete}
```

Comment:
`delete` is a reserved keyword, could this cause problems in some browsers (looking at you, IE). I haven't checked though.

Reply:
Good point.
Changed to createItem, editItem, deleteItems and findItems
Which is convenient becuase it also illustrates which of the methods support handling multiple items.